Resources/Prototypes/Entities/Structures/Walls/asteroid.yml

```diff
@@ -52,6 +64,47 @@
     oreChance: 0.33
     oreRarityPrototypeId: RandomOreDistributionStandard
 
+
+- type: entity
+  id: IronRock
+  parent: AsteroidRock
+  name: ironrock
+  suffix: Low Ore Yield
+  description: A rocky asteroid.
```

Comment:
Should also localize this to be consistent with all other walls. See #18827. Same for the other IronRockMining. Worth removing the localizations no longer in use too.

Reply:
Strike the above remark, should #18881 be merged.